tasks/db_deploy/migrate_db.py

```diff
@@ -5,8 +5,9 @@
 """
 from typing import Dict
 from orca_sql import *
-from orca_shared.shared_db import get_admin_connection, logger
+from orca_shared.shared_db import get_admin_connection, logger, retry_operational_error
```

Comment:
Remove `retry_operational_error` library is not needed in this file

Reply:
removed